src/storage/txn/actions/acquire_pessimistic_lock.rs

```diff
@@ -480,6 +497,16 @@ pub mod tests {
         assert_eq!(lock.lock_type, LockType::Pessimistic);
     }
 
+    pub fn lock_must_not_exist<E: Engine>(
```

Comment:
```suggestion
    pub fn must_no_lock<E: Engine>(
```
To keep it consistent with other functions in the tests like `must_get_value`, `must_prewrite`.

Reply:
done